docs/index.md

```diff
@@ -8,12 +8,32 @@ nav_order: 1
 GeoBlazor is a set of easy-to-use Razor Components for creating a Blazor Server, Blazor Wasm, or Blazor Hybrid (MAUI) 
 application with the full power of ArcGIS.
 
+## Documentation Pages
+- [Getting Started](pages/gettingStarted) - How to set up your first GeoBlazor application
+- [Authentication](pages/authentication) - Advanced options for authenticating with ArcGIS
+- [API Documentation](pages/classes/index) - Detailed technical specification for GeoBlazor types and classes
+
 ## Useful Links
-- [Home Page](https://www.geoblazor.com)
-- [Live Samples](https://samples.geoblazor.com)
-- [GitHub Repository](https://github.com/dymaptic/geoblazor)
-- [Nuget Package](https://www.nuget.org/packages/dymaptic.GeoBlazor.Core)
-- [Discord Server](https://discord.gg/hcmbPzn4VW) - for discussions, Q&A
+- [Home Page](https://www.geoblazor.com) - main website
+- [Live Samples](https://samples.geoblazor.com) - interactive demo pages
```

Comment:
Need to be consistent on capitalization. The first 3 lines start with capital letters but these last 2 don't.

Reply:
Same applies to the sections below